jekyll/_cci2/orb-intro.md

```diff
@@ -157,16 +157,21 @@ jobs:
 ## Orbs page in the CircleCI app
 {: #orbs-view}
 
-The Orbs page gives users visibility into the orbs their organization has created. Users can review the orb type (public or private), orb usage (how many times the orb is used across all configurations), latest version, and description directly from the list view on the page.
+The orbs page in the CircleCI web app is not currently available on CircleCI server.
+{: class="alert alert-warning"}
+
+To access the orbs page in the web app, navigate to **Organization Settings** and select **Orbs** from the sidebar.
 
-To access the Orbs page, navigate to Organization Settings in the app.
+The orbs page list orbs created within your organization. You can view orb type (public or private), orb usage (how many times the orb is used across all configurations), latest version, and description.
```

Comment:
Should this be "The orbs page _lists_ orbs..."? Just a suggestion to break up the page, but we could also list it out like

> You can view:
- orb type (public or private)
- orb usage (how many times the orb is used across all configurations)
- latest version
- description